modules/installation-creating-aws-control-plane.adoc

```diff
@@ -0,0 +1,208 @@
+  },
+  {
+    "ParameterKey": "InternalApiTargetGroupArn", <33>
+    "ParameterValue": "arn:aws:elasticloadbalancing:<zone>:269733383066:targetgroup/<dns_stack>-Inter-1Q3IAOSC8ZNNM/8e85701acf15632d" <34>
+  },
+  {
+    "ParameterKey": "InternalServiceTargetGroupArn", <35>
+    "ParameterValue": "arn:aws:elasticloadbalancing:<zone>:269733383066:targetgroup/<dns_stack>-Inter-1INRIWR8L3627/4cd9e764bb418f47" <36>
+  }
+]
+----
+<1> The name for your cluster infrastructure that is encoded in your Ignition
+config files for the cluster.
+<2> Specify the infrastructure name that you extracted from the Ignition config
+file metadata, which has the format `<cluster-name>-<random-string>`.
+<3> Current{op-system-first} AMI to use for the control plane machines.
+<4> Specify an `AWS::EC2::Image::Id` value.
+<5> Whether or not to perform DNS etcd registration.
+<6> Specify `yes` or `no`. If you specify `yes`, you must provide Hosted Zone
+information.
+<7> The Route53 private zone ID to register the etcd targets with.
+<8> Specify the `PrivateHostedZoneId` value from the output of the
+CloudFormation template for DNS and load balancing.
+<9> The Route53 zone to register the targets with.
+<10> Specify `<cluster_name>.<domain_name>` where `<domain_name>` is the Route53
+base domain that you used when you generated the Ignition
+config files for the cluster. Do not include the trailing period (.) that is
+displayed in the AWS console.
+<11> A subnet, preferably private, to launch the control plane machines on.
+<12> Specify a subnet from the `PrivateSubnets` value from the output of the
+CloudFormation template for DNS and load balancing.
+<13> A subnet, preferably private, to launch the control plane machines on.
+<14> Specify a subnet from the `PrivateSubnets` value from the output of the
+CloudFormation template for DNS and load balancing.
+<15> A subnet, preferably private, to launch the control plane machines on.
+<16> Specify a subnet from the `PrivateSubnets` value from the output of the
+CloudFormation template for DNS and load balancing.
+<17> The master security group ID to associate with master nodes.
+<18> Specify the `MasterSecurityGroupId` value from the output of the
+CloudFormation template for the security group and roles.
+<19> The location to fetch control plane Ignition config file from.
+<20> Specify the generated Ignition config file location,
+`https://api.<cluster_name>.<domain_name>:22623/config/master`.
+<21> The base64 encoded certificate authority string to use.
+<22> Specify the value from the `master.ign` file that is in the installation
+directory. This value is the long string with the format
+`data:text/plain;charset=utf-8;base64,ABC...xYz==`.
+<23> The IAM profile to associate with master nodes.
+<24> Specify the `MasterInstanceProfile` parameter value from the output of
+the CloudFormation template for the security group and roles.
+<25> The type of AWS instance to use for the control plane machines.
+<26> Allowed values:
```

Comment:
Latest generation of EC2 instances aren't allowed (m5, c5, r5).  It might be interesting to allow AMD based instances as well.

Reply:
Same as above.